src/Api/PubnubApi/Security/SecureMessage.cs

```diff
@@ -68,12 +68,10 @@ public List<object> HistoryDecodeDecryptLoop<T>(PNOperationType type, List<objec
                             {
                                 status.AffectedChannelGroups.AddRange(channelGroups);
                             }
-
-                            errorCallback.OnResponse(default(T), status);
```

Comment:
Ohhh, I missed this during the previous change ;o?

Reply:
this is fine!